examples/ZED-F9P/Example6_GetVal/Example6_GetVal.ino

```diff
@@ -32,27 +32,31 @@ long lastTime = 0; //Simple local timer. Limits amount if I2C traffic to Ublox m
 void setup()
 {
   Serial.begin(115200);
-  while (!Serial); //Wait for user to open terminal
-  Serial.println("Ublox getVal example");
+  while (!Serial)
+    ; //Wait for user to open terminal
+  Serial.println("u-blox getVal example");
 
   Wire.begin();
   Wire.setClock(400000); //Increase I2C clock speed to 400kHz
 
   if (myGPS.begin() == false) //Connect to the Ublox module using Wire port
   {
-    Serial.println(F("Ublox GPS not detected at default I2C address. Please check wiring. Freezing."));
-    while (1);
+    Serial.println(F("u-blox GPS not detected at default I2C address. Please check wiring. Freezing."));
+    while (1)
+      ;
   }
 
   myGPS.enableDebugging(); //Enable debug messages over Serial (default)
   //myGPS.enableDebugging(SerialUSB); //Enable debug messages over Serial USB
 
-  uint8_t currentI2Caddress = myGPS.getVal8(0x20510001);
+#define UBLOX_CFG_I2C_ADDRESS 0x20510001
```

Comment:
UBLOX_CFG_I2C_ADDRESS is defined in u-blox_config_keys.h. I think you'll get re-definition warnings / errors here?

Reply:
Good catch. Left over from testing. Removed.